<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>827</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>827</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>273</y>
      <width>511</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="choicesLayout">
     <item>
      <widget class="QLabel" name="choicesBox">
       <property name="text">
        <string>Choices Box</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="choicesSystem"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>410</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="resultLayout">
     <item>
      <widget class="QLabel" name="resultLabel">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="answerResult"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>811</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="systemLayout">
     <item>
      <widget class="QLabel" name="system">
       <property name="text">
        <string>System</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGraphicsView" name="counterpointSystem">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::NoDrag</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="newExerciseButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Create New Exercise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkAnswerButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Check Answer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseSymbol">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Note</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
